### PR TITLE
Problem: snmp-ups segfaults

### PR DESCRIPTION
**Pull request by bstepankova**
Solution:  base_snmp_template_index() prevented from processing empty argument

Signed-off-by: Barbora Stepankova <BarboraStepankova@eaton.com>

### CODE REVIEW

File: drivers/snmp-ups.c [hunk 1]
```diff
@@ -2225,21 +2225,23 @@ bool_t daisychain_init()
...
 
+    /* Finally, compute and store the base OID index and NUT offset */
+    su_info_p = su_find_info("device.model");
+    if (su_info_p != NULL) {
+        device_template_index_base = base_snmp_template_index(su_info_p);
+        upsdebugx(1, "%s: device_template_index_base = %i", __func__, device_template_index_base);
+        device_template_offset = device_template_index_base - 1;
+        upsdebugx(1, "%s: device_template_offset = %i", __func__, device_template_offset);
+    }
+
```
Comment (jimklimov) [inline drivers/snmp-ups.c]: Extra blank lines ;)

Perhaps also add something like 
````
else { upsdebugx(1, "No device.model entry found, device template not customized for daisy chain");  }
````

to see that we skipped this block of settings (and @aquette : what should be the default values in this case? -1? 0? something else? does the other code have a concept of these not having been determined -- maybe as part of `daisychain_enabled == FALSE` handling?)

Finally, I wonder how we might get into situation of responding device not having a model set - maybe we inspect the `daisychain_init()` a bit too early in driver startup?

**Comment by jimklimov** [comment]
Merging to test how this variant behaves in our master images

**Comment by aquette** [comment]
@bstepankova  and @jimklimov : was this only for master and not 1.3.x?
Also, I see no matching PR in NUT upstream...

**Comment by bstepankova** [comment]
@aquette  we decided not to merge it to 1.3.1 until Yves says it works.

**Pull request merged**